e2e/imgbuild/imgbuild.go

```diff
@@ -154,6 +159,511 @@ func badPath(t *testing.T) {
 	}
 }
 
+func buildMultiStageDefinition(t *testing.T) {
+	tmpfile, err := ioutil.TempFile(testenv.TestDir, "testFile-")
```

Comment:
This `tmpfile` stuff looks like it would be good to move to a function for reuse!